frontend/src/components/UserForm.jsx

```diff
@@ -87,6 +99,37 @@ const notToSelect = ["Tanzania"];
       alert('Please select your education level');
       return;
     }
+
+    try{
+      const response = await fetch("http://localhost:3000/api/users", {
```

Comment:
Kwanini hapa tunatumia port 3000? Our backend si ina run kwenye port 5000

Reply:
Also I will suggest the backend url ikae kwenye environment variable...meaning tengeneza .env file ya frontend..(iwe kwenye rootfolder ya frontend just like what we have under backend) the and API_URL="http://localhost:5000"

then at the file your use fetch will be like below:
```suggestion
      const response = await fetch(`${process.env.API_URL}/api/users`, {
```

You may need to install the package called loadenv